apps/prairielearn/src/pages/enroll/enroll.sql

```diff
@@ -1,7 +1,7 @@
 -- BLOCK select_course_instances
 SELECT
   c.short_name || ': ' || c.title || ', ' || ci.long_name AS label,
-  c.short_name || ', ' || ci.short_name AS short_label,
+  c.short_name || ', ' || ci.long_name AS short_label,
```

Comment:
IMO we should never be exposing course instance short names to students.

Reply:
I'm on board with that.